src/homework11/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,59 @@
+package homework11;
+import java.util.HashMap;
+import java.util.Map;
+
+//1. Створити клас CoffeeOrderBoard. Клас є цифровим поданням черги замовлень у кав'ярні.
+//Цей клас містить список замовлень.
+public class CoffeeOrderBoard {
+
+    public final Map<Integer, Order> orderBoard = new HashMap<>();
+    public static int orderNumber = 0;
```

Comment:
Why are they public? Incapsulation principles are broken here. 
According to the task, you must have the entity called `CoffeeOrderBoard` with 4 methods - `add(Order order)`, `deliver()`,  `deliver(int orderNumber)` and `draw()`, nothing more. How it works, where and how it store the data - is not a "programmer-user" (the person who will use this class) business.